libqtile/widget/wlan.py

```diff
@@ -71,7 +73,11 @@ def poll(self):
             essid, quality = get_status(self.interface)
             disconnected = essid is None
             if disconnected:
-                return self.disconnected_message
+                statfile = open("/sys/class/net/"+self.ethint+"/operstate", "r")
```

Comment:
Could you use 
```python
with open(f"/sys/class/net/{self.ethint}/operstate", "r")) as statfile:
    ... rest of code indented here ...
```

here? That makes sure the file is closed at the end and makes sure f strings are used

Reply:
Done!